app/portainer/views/groups/create/createGroupController.js

```diff
@@ -31,6 +31,12 @@ function ($q, $scope, $state, GroupService, EndpointService, TagService, Notific
     });
   };
 
+  $scope.onCreateTag = async function onCreateTag(tagName) {
+    const tag = await TagService.createTag(tagName);
+    $scope.availableTags = $scope.availableTags.concat(tag);
+    $scope.model.TagIds = $scope.model.TagIds.concat(tag.Id);
+  }
```

Comment:
Wrap with non async function. The main goal for this wrapper is to trigger a digest cycle when the async function returns. See #2945 for more details.

Also, protect the await with a try/catch closure and show a notification if error occurs